feat(cb2-19777): test and tech records sections

tomevs88 opened the pull request:
## Ticket title

_One line description_
[CB2-19777](https://dvsa.atlassian.net/browse/CB2-19777)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Delete branch after merge


[CB2-19777]: https://dvsa.atlassian.net/browse/CB2-19777?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

tomevs88 committed:
feat(cb2-19777): test and tech records sections

matthew2564 commented on src/app/forms/custom-sections-v2/test-records/test-records.component.ts:
```diff
@@ -0,0 +1,131 @@
+
+	getTestTypeResults(testResult: TestResultModel): string {
+		return testResult.testTypes.map((t) => t.testResult).join(',');
+	}
```
Are these used / needed?

pbardy2000 committed:
fix(CB2-19844): fix link behaviour (#2177)

* fix(CB2-19844): fix link behaviour

* fix(CB2-19844): add id

* fix(CB2-19844): fix link styling

* fix(CB2-19844): add break all to non-link lines

tomcrawleyy committed:
feat(CB2-19861): Small TRL required fields (#2176)

* feat(CB2-19861): initial commit

* feat(CB2-19861): remove subclass changes

pbardy2000 committed:
Feature/cb2 19856 2 (#2181)

* feat(CB2-19856): initial commit

* feat(CB2-19856): create mode fixes

* feat(CB2-19856): filters on amend

---------

Co-authored-by: Thomas Crawley <thomas.crawley@dvsa.gov.uk>
# Conflicts:
#	src/app/features/tech-record/components/vehicle-technical-record-wrapper/vehicle-technical-record-v2/vehicle-technical-record-v2.component.html
#	src/app/features/tech-record/components/vehicle-technical-record-wrapper/vehicle-technical-record-v2/vehicle-technical-record-v2.component.ts

pbardy2000 committed:
fix(CB2-19976): fix styling (#2180)

* fix(CB2-19976): fix styling

* fix(CB2-19976): add id to back button

tomevs88 committed:
feat(cb2-19777): tab in issue

tomevs88 committed:
feat(cb2-19777): tab in issues

tomevs88 committed:
feat(cb2-19777): tab in issues
